README.md

```diff
@@ -517,7 +525,15 @@ Check the [ImageOptim](https://github.com/toy/image_optim#configuration) to get
 | `optim`                           | ✔             |
 | `@optim`                          | ✗             |
 
-***By default `@optim` will use the default `jekyll`, otherwise you can provide `optim=preset` and have it used that preset.  ImageOptim provides advanced, and default as their default presets, you can define your own preset via Jekyll Assets configuration listed above.***
+***By default `@optim` will use the default configuration set `jekyll`, otherwise you can provide `optim=preset` and have it used that preset.  ImageOptim provides advanced, and default as their default presets, you can define your own preset via Jekyll Assets configuration listed above.***
```

Comment:
Your wording is confusing to me because it's not a set, it's a dictionary (a hash).

Reply:
Well it is more or less the wording of the code, a "preset" is a short for "pre-configured set"

Reply:
Preset means to set ahead of time, or to set it beforehand.  Not that it's a "preconfigured set"